ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -82,6 +85,40 @@
       />
     </Hds::ButtonSet>
   </div>
+  {{#if this.showMultipleModalsMessage}}
+    <Hds::Modal
+      id="multiple-message-modal"
+      @size="large"
+      @color="warning"
+      @onClose={{fn (mut this.showMultipleModalsMessage) false}}
+      data-test-modal="preview modal"
+      as |M|
+    >
+      <M.Header data-test-modal-title="Warning: more than one modal">
+        Warning: more than one modal
+      </M.Header>
+      <M.Body data-test-modal-body="Warning: more than one modal">
+        You have an active modal configured and are trying to create another one. It is recommended to
+        <b>avoid having more than one modal</b>
```

Comment:
Is this per authenticated state? Might be helpful to point out if so

Reply:
This is for both auth and unauth states :)

Reply:
What i mean is, is this true if I have an unauth'd modal but am setting up an auth'd modal? Seems like this should only happen if there are already modals of the same auth type that I'm attempting to set up.

Reply:
Yup it should be only apply for the same auth type :)